frontend/assets/stylesheets/components/nav/_nav.scss

```diff
@@ -150,6 +151,60 @@ $global-nav-height: 36px;
     }
 }
 
+.nav--global--subnav {
```

Comment:
Think we might be starting to get a BEM code-smell here. I'm not sure these need to be modifiers. Double modifier classes in BEM always feel like a red flag to me, but this is following what's already there so it's OK. Might be something to come back to as a later bit of refactoring once this goes out, doesn't feel like too big a deal yet.

My point is, what colour should we paint this bikeshed?